framework/auth/oauth_scopes.py

```diff
@@ -41,6 +41,10 @@ class CoreScopes(object):
     APPLICATIONS_READ = 'applications_read'
     APPLICATIONS_WRITE = 'applications_write'
 
+    NODE_LOG_READ = 'nodes.logs_read'
+
+    NULL = 'null'
```

Comment:
What do you mean by "always read only"? I'd think read/write access would be determined by what kind of view class you used.

If you're referring to "can be written for internal use, but not third party users", then still no. The way to keep it out of a write scope is to keep a correspponding core scope entry, but omit it from write access in `ComposedScopes`